### PR TITLE
removing required FASTA reference input in SV discovery pipeline

### PR DESCRIPTION
**Pull request by SHuang-Broad**
(for its dict) for sorting variants in output VCF, now using header in input SAM/BAM

### CODE REVIEW

**Comment by SHuang-Broad** [comment]
@TedBrookings will you be willing to take a look?

**Comment by TedBrookings** [comment]
Yeah, I've done a quick skim and it looks like I should be able to review.

**Comment by SHuang-Broad** [comment]
Thanks!

File: src/test/java/org/broadinstitute/hellbender/tools/spark/sv/integration/DiscoverVariantsFromContigAlignmentsSAMSparkIntegrationTest.java [hunk 1]
```diff
@@ -35,14 +35,13 @@
...
                     " -I " + SVIntegrationTestDataProvider.TEST_CONTIG_SAM +
-                    " -O " + outputDir + "/variants.vcf" +
-                    " --fastaReference " + SVIntegrationTestDataProvider.reference;
+                    " -O " + outputDir + "/variants.vcf";
         }
 
     }
 
     @DataProvider(name = "discoverVariantsFromContigAlignmentsSparkIntegrationTest")
-    public Object[][] createTestData() throws IOException {
```
Comment (TedBrookings) [inline src/test/java/org/broadinstitute/hellbender/tools/spark/sv/integration/DiscoverVariantsFromContigAlignmentsSAMSparkIntegrationTest.java]: Maybe due to my lack of java knowledge, but why was the "throws IOException" removed?

Comment (SHuang-Broad) [reply src/test/java/org/broadinstitute/hellbender/tools/spark/sv/integration/DiscoverVariantsFromContigAlignmentsSAMSparkIntegrationTest.java]: For record keeping: my version of IDE indicates the function (or actually the function it calls into) now doesn't throw `IOException` any more, so I removed it.

**Pull request merged**